python/ray/train/tests/test_gpu.py

```diff
@@ -349,6 +349,10 @@ def test_tune_tensorflow_mnist_gpu(ray_start_4_cpus_2_gpus):
     tune_tensorflow_mnist(num_workers=2, use_gpu=True, num_samples=1)
 
 
+def test_concurrent_tune_tensorflow_mnist_gpu(ray_start_4_cpus_2_gpus):
+    tune_tensorflow_mnist(num_workers=1, use_gpu=True, num_samples=2)
```

Comment:
The change was made to `train.torch`, so tensorflow tests are not actually testing the changes.